web/client/components/misc/panels/DockPanel.jsx

```diff
@@ -52,7 +53,8 @@ module.exports = withState('fullscreen', 'onFullscreen', false)(
         footer,
         children,
         onFullscreen = () => {},
-        fixed = false
+        fixed = false,
+        hideHeader
```

Comment:
I would call it showHeader with the logic inverted, but this is subjective, isn't it? @offtherailz @mbarto

Reply:
I would leave it as is